contracts/implementations/nodes/SyntheticDevice.sol

```diff
@@ -82,6 +82,96 @@ contract SyntheticDevice is AccessControlInternal {
 
     // ***** Interaction with nodes *****//
 
+    /**
+     * @notice Mints a list of synthetic devices and link them to vehicles
+     * To be called for existing vehicles already connected
+     * @dev Caller must have the admin role
+     * @param data input data with the following fields:
+     *  integrationNode -> Parent integration node id
```

Comment:
Thinking aloud

* I wonder if it would clean things up to only allow batch minting under one integration at a time.
* Does it help us at all to drop the requirement for an SD signature?

Reply:
- About the integration, it would be cleaner. We would have to make sure the data is correct in advance in both cases. So I would take the simpler way
- It would save some gas. Also could move for example `data.vehicleNode` to a local variable, since we got some room in the stack. I would love to remove this signature.